qbittorrent/umbrel-app.yml

```diff
@@ -0,0 +1,30 @@
+manifestVersion: 1
+id: qbittorrent
+category: Networking
+name: qBittorrent
+version: "4.5.2"
+tagline: A fast, easy and free BitTorrent client
+description: >-
+  The qBittorrent project aims to provide an open-source software alternative to µTorrent.
+  
+  Additionally, qBittorrent runs and provides the same features on all major platforms (FreeBSD, Linux, macOS, OS/2, Windows).
+  
+  Bittorrent is based on the Qt toolkit and libtorrent-rasterbar library.
+  
+  ⚠️ qBittorrent downloads torrents over the Clearnet, not Tor.
+developer: qBittorrent
+website: https://qbittorrent.org/
+dependencies: []
+repo: https://github.com/qBittorrent/qBittorrent
+support: https://github.com/qBittorrent/qBittorrent/issues
+port: 8080
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: ""
+defaultUsername: "admin"
+defaultPassword: "adminadmin"
+torOnly: false
+permissions:
+  - STORAGE_DOWNLOADS
```

Comment:
You get to add yourself as `submitter` here! And also add a `submission` field with the url of this PR as the value.
See Thunderhub app for example:
https://github.com/getumbrel/umbrel-apps/blob/91be88379b12c79a047e30c0c311a60aaf13c932/thunderhub/umbrel-app.yml#L38-L39